### PR TITLE
data_structure_and_data_cleaning

### PR DESCRIPTION
**Pull request by northernleaf**
Submit CC for group 36

### CODE REVIEW

**Review by bl2788 — CHANGES_REQUESTED** [review]
please remove the setup label in your setup trunk

**Comment by bojum** [comment]
@northernleaf Don't worry about this. This is as a note to maintainers.

The PR got this error message from the build failure:

Installing package dependencies
0.38s$ Rscript -e 'bookdown::render_book("index.Rmd")'
Error in yaml::yaml.load(..., eval.expr = TRUE) : 
  Parser error: while parsing a flow sequence at line 8, column 12 did not find expected ',' or ']' at line 28, column 3
Calls: <Anonymous> ... <Anonymous> -> resolve -> <Anonymous> -> yaml_load -> <Anonymous>
Execution halted
The command "Rscript -e 'bookdown::render_book("index.Rmd")'" exited with 1.

If any maintainers can look into it.

**Comment by jtr13** [comment]
The problem is a missing comma in _bookdown.yml

File: _bookdown.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ rmd_files: [
   # tutorial
   'carto_tutorial.Rmd',
   'a_tutorial_of_shiny.Rmd',
+  'data_structure_and_data_cleaning_tutorial.Rmd'
```
Comment (jtr13) [inline _bookdown.yml]: missing comma

**Comment by skyetim** [comment]
🎉 Thanks for your contribution! It has been merged!

**Pull request merged**